components/BottomNavigation/README.md

```diff
@@ -181,3 +181,82 @@ id<MDCTypographyScheming> typographyScheme = [[MDCTypographyScheme alloc] init];
 ```
 <!--</div>-->
 
+## Accessibility {#a11y}
+
+To help ensure your bottom navigation item is accessible to as many users as possible, please
+be sure to review the following recommendations:
+
+### `-accessibilityLabel`
+The label will be the title of the UITabBarItem. Currently you can't set this to a custom value.
+
+### Set `-accessibilityValue` 
+
+Set an appropriate
+[`accessibilityValue`](https://developer.apple.com/documentation/uikit/uiaccessibilityelement/1619583-accessibilityvalue)
+value if your item has a badge value. For example, an item with an inbox icon with a badge value for how many
+emails are unread. You should explicitly set the `accessibilityValue` when the badge value doesn't provide enough context. For example, in an inbox example simply having
+
+#### Objective-C
+```objc
+accessibilityValue = @"10";
+``` 
+#### Swift
+```swift
+accessiblityValue = "10"
+```
+doesn't provide enough context, instead the accessibility value should explain what the badge value symbolizes.
```

Comment:
I think this is very hard to read, especially for people using screen readers. Can you please do a little copyediting/clean-up so we don't have 2 code blocks in the middle of the sentence?